### ISSUE TITLE
Review WCAG reference for standard 05.09

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#48281]
## Description
[See Slack thread from 10/17/2022](https://dsva.slack.com/archives/GNF5GSD9N/p1665774176386319)

**Comment by briandeconinck** [comment]
No longer applicable. The changes to the Experience Standards make them less "WCAG-lite" and there's not likely to be a one-to-one relationship between WCAG and our standards the way there is now.
